include/exadg/postprocessor/pointwise_output_generator_base.cpp

```diff
@@ -141,13 +140,13 @@ PointwiseOutputGeneratorBase<dim, Number>::setup_base(
       auto group = hdf5_file->create_group("PhysicalInformation");
       add_time_dataset(group, "Time");
     }
-  }
+
 #else
-  (void)triangulation_in;
-  (void)mapping_in;
-  (void)pointwise_output_data_in;
-  AssertThrow(false, dealii::ExcMessage("deal.II is not compiled with HDF5!"));
```

Comment:
@jh66637 isn't the main point of the present PR that this assert is currently always triggered independently of whether we want to use this postprocessing routine or not?

I understood the present PR in the way that this line currently enforces HDF5 to be a mandatory TPL for ExaDG. And this way I see a link to issue #419.

Reply:
I agree, but I can not reproduce it on my laptop.

Reply:
It looks like you need to find out in more detail what your local setup is doing exactly.